README.md

```diff
@@ -177,6 +170,22 @@ root@9feae5e5e001:/src/conjur/cucumber/api# cucumber
 root@9feae5e5e001:/src/conjur# cucumber -r cucumber/api/features/support -r cucumber/api/features/step_definitions cucumber/api/features/resource_list.feature
 ```
 
+## Documentation site
+
+This repository also contains the entire source code for the [Conjur
+documentation website][docs]. For instructions on how to work on the
+site locally, visit the [docs README][docs-readme].
+
+Or in brief:
+
+```sh-session
+$ docker-compose up -d docs
```

Comment:
Is the `docker-compose build --pull docs` step no longer necessary?

Reply:
Also, is it worth including the command for building the API docs here? If someone just follows these brief instructions they will not have any API docs.

Reply:
The build step will kick off automatically if it hasn't before. If you want to re-download all the images and re-build everything, then you need to `build --pull docs` - but that's rarely beneficial and takes a long time, so I prefer leaving that as an advanced exercise rather than putting it in the instructions.